Dockerfile

```diff
@@ -1,5 +1,6 @@
 FROM node:16
 
+USER node
```

Comment:
@jgerigmeyer Unrelated, but I think we need something like this in our Dockerfiles to prevent them from running as root. The `node` user has ID 1000, which is my main user in WSL, so no problem there to build and serve the site.

Could you check if this doesn't cause trouble in Mac OS?

Reply:
@jerivas Unfortunately this change does cause an error for me:

```
------
 > [dev_container_auto_added_stage_label 4/4] RUN yarn install:
#0 0.702 ➤ YN0000: ┌ Resolution step
#0 0.866 ➤ YN0000: └ Completed
[2023-06-10T13:44:09.103Z] 
#0 0.867 ➤ YN0000: ┌ Fetch step
#0 1.022 ➤ YN0013: │ sass@npm:1.62.1 can't be found in the cache and will be fetched from the remote registry
#0 2.487 ➤ YN0001: │ Error: sass@npm:1.62.1: EACCES: permission denied, copyfile '/tmp/xfs-e276237b/archive.zip' -> '/app/.yarn/cache/sass-npm-1.62.1-c16d65fd28-1b1b3584b3.zip-2311b2c4377dc4b3.tmp'
#0 2.487 ➤ YN0000: └ Completed in 1s 620ms
#0 2.487 ➤ YN0000: Failed with errors in 1s 786ms
------
```

Reply:
Reverted for now.